app/forms/claim/provider_form.rb

```diff
@@ -0,0 +1,47 @@
+class Claim::ProviderForm < ApplicationForm
+  attr_accessor :id, :provider_id, :school
+
+  validate :validate_provider
+
+  def persist
+    updated_claim.save!
+  end
+
+  def providers
+    @providers ||= Provider.where(name: [
+      "Best Practice Network",
```

Comment:
Is it worth creating a constant in Provider, and maybe a scope, to always get these providers?

Reply:
Sure